turtlebot3: #namespace
    task_and_robot_environment_name: 'TurtleBot3Navigation-v0'
    ros_ws_abspath: "/home/nicholas/project_cws"
    running_step: 0.04 # amount of time the control will be executed
    pos_step: 0.016     # increment in position for each command
    
    #qlearn parameters
    alpha: 0.5
    gamma: 0.7
    epsilon: 0.995
    epsilon_discount: 0.995 #0.999
    nepisodes: 2000
    nsteps: 200
    running_step: 0.06 # Time for each step